pydis_site/apps/api/migrations/0053_merge_20200114_1559.py

```diff
@@ -0,0 +1,14 @@
+# Generated by Django 2.2.8 on 2020-01-14 15:59
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('api', '0049_deletedmessage_attachments'),
+        ('api', '0052_add_whitelist_unique_constraint'),
+    ]
+
+    operations = [
+    ]
```

Comment:
What is the purpose of this migration if it has no operations?

Reply:
To merge two heads, and restore linear order to the migration history.